pkg/services/govmomi/vcenter/clone.go

```diff
@@ -453,3 +461,55 @@ func getNetworkSpecs(ctx *context.VMContext, devices object.VirtualDeviceList) (
 
 	return deviceSpecs, nil
 }
+
+func createPCIPassThroughDevice(deviceKey int32, backingInfo types.BaseVirtualDeviceBackingInfo) types.BaseVirtualDevice {
```

Comment:
Let's move this func to the very bottom of the file, because its a helper for the ones below.